Test cases/closures.kt

```diff
@@ -44,6 +44,9 @@ internal fun g2(a: Int = 0, closure: () -> Int) {
 internal fun f3(closure: () -> Unit) {
 }
 
+internal fun f4(closure: () -> Unit) {
```

Comment:
Looking good! Could you add a comment above `f4` saying what it's testing for future reference? Just like the `f3` and `g1` functions above it.